packages/features/teams/pages/team-profile-view.tsx

```diff
@@ -309,57 +238,15 @@ const TeamProfileForm = ({ team }: TeamProfileFormProps) => {
       form={form}
       handleSubmit={(values) => {
         if (team) {
-          const variables = {
-            name: values.name,
-            slug: values.slug,
-            bio: values.bio,
-            logo: values.logo,
-          };
-          objectKeys(variables).forEach((key) => {
-            if (variables[key as keyof typeof variables] === team?.[key]) delete variables[key];
-          });
+          const variables = Object.fromEntries(
+            Object.entries({ name: values.name, slug: values.slug }).filter(
+              ([key, value]) => value !== team[key as keyof typeof team]
+            )
+          );
```

Comment:
You might be able to use `import { pickyBy } from 'lodash';` here.

I think it reads slightly better - what do you think?

```suggestion
          const variables = pickBy(
            { name: values.name, slug: values.slug },
            (value, key) => value !== team[key as keyof typeof team]
          );
```

Reply:
Definitely reads sweeter... just wondering is it worth installing lodash for this single use? Or maybe something to use more consistently going forward?

Reply:
Oh I thought it was installed already but it's not for the features packages